dockerfiles/run/guice/cassandra-ldap/Dockerfile

```diff
@@ -28,4 +28,6 @@ ADD destination/conf /root/conf
 VOLUME /logs
 VOLUME /root/conf
 
-ENTRYPOINT java -Dlogback.configurationFile=/root/conf/logback.xml -Dworking.directory=/root/ -jar james-server.jar 
+ENV JVM_OPTIONS="-XX:-PrintGC"
+
+ENTRYPOINT java -Dlogback.configurationFile=/root/conf/logback.xml -Dworking.directory=/root/ $JVM_OPTIONS -jar james-server.jar
```

Comment:
this should not work with options containing spaces. Do we care?

Reply:
oh you missed my point, `--env JVM_OPTIONS="-Xms256m -Xmx2048m"` is passing two options `-Xms256m` and `-Xmx2048m`. My point is about having one option containing a space (for example for a path).

Reply:
Sorry, I don't get you...
Can you give me an example of valid jvm options you are thinking of?

Reply:
`-Xloggc:/path with spaces/gc.log`

Reply:
Do we really want to handle such cases?

Reply:
That's why I started by " Do we care?" ;)

Reply:
My opinion is: "no"
But I'm not the PO ;)

Reply:
can you just check if dockerFile handles "${JVM_OPTS}" syntax? (that's a shell hack)

Reply:
Good catch, it's supported in Dockerfiles.